extensions/amp-analytics/0.1/events.js

```diff
@@ -151,6 +151,18 @@ function isAmpStoryTriggerType(triggerType) {
   return triggerType.startsWith('story');
 }
 
+/**
+ * Assert that the selectors are all unique
+ * @param {!Array<string>|string} selectors
+ */
+function assertUniqueSelectors(selectors) {
+  userAssert(
+    !isArray(selectors) || new Set(selectors).size === selectors.length,
+    'Cannot have duplicate selectors in selectors list: %s',
+    selectors
+  );
```

Comment:
My bad, it looks like constructing like `new Set(iterable)` [is not supported on Internet Explorer.](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Set/Set#browser_compatibility) We should add a clause to gracefully continue in this case.

```suggestion
  const unique = isArray(selectors) ? new Set(selectors) : null;
  userAssert(
    unique === null ||
      unique.size === selectors.length ||
      // TODO(https://go.amp.dev/issue/34453): Internet Explorer does not
      // support the  constructor `new Set(iterable)`, so it would be empty.
      // We assume it's ok to continue in this case.
      // We should remove the following clause once we drop IE support.
      (!IS_ESM && unique.size === 0),
    "Cannot have duplicate selectors in selectors list: %s",
    selectors
  );
```

(`IS_ESM` is a flag pertaining to build output. It's "magically" defined and you don't need to import it or set it.)